wazimap_ng/cache.py

```diff
@@ -65,55 +73,79 @@ def last_modified_point_updated(request, profile_id, profile_category_id=None, t
 
     return last_modified(request, profile_id, key)
 
+
 ########### Signals #################
 def update_profile_cache(profile):
     logger.info(f"Updating profile cache: {profile}")
     key = profile_key % profile.id
     cache.set(key, datetime.now())
 
+
 def update_point_cache(category):
     profile = category.profile
     key1 = location_key % (profile.id, category.id)
 
     logger.debug(f"Set cache key (category): {key1}")
     cache.set(key1, datetime.now())
 
+
 @receiver(post_save, sender=ProfileIndicator)
 def profile_indicator_updated(sender, instance, **kwargs):
     update_profile_cache(instance.profile)
 
+
 @receiver(post_save, sender=ProfileHighlight)
 def profile_highlight_updated(sender, instance, **kwargs):
     update_profile_cache(instance.profile)
 
+
 @receiver(post_save, sender=IndicatorCategory)
 def profile_category_updated(sender, instance, **kwargs):
     update_profile_cache(instance.profile)
 
+
 @receiver(post_save, sender=IndicatorSubcategory)
 def profile_subcategory_updated(sender, instance, **kwargs):
     update_profile_cache(instance.category.profile)
 
+
 @receiver(post_save, sender=ProfileKeyMetrics)
 def profile_keymetrics_updated(sender, instance, **kwargs):
     update_profile_cache(instance.profile)
 
+
 @receiver(post_save, sender=Profile)
 def profile_indicator_updated(sender, instance, **kwargs):
     update_profile_cache(instance)
 
+
 @receiver(post_save, sender=Location)
 def point_updated_location(sender, instance, **kwargs):
     update_point_cache(instance.category)
 
+
 @receiver(post_save, sender=Category)
 def point_updated_category(sender, instance, **kwargs):
     update_point_cache(instance)
 
+
+@receiver(post_save, sender=Indicator)
+def indicator_updated(sender, instance, **kwargs):
+    indicator_id = instance.id
+    profiles_to_invalidate_cache = Profile.objects.filter(
```

Comment:
as @arghyaiitb pointed out we should make this distinct. but then we're ready to go.